packages/okta-react/test/e2e/harness/e2e/page-objects/app.po.js

```diff
@@ -20,15 +20,15 @@ export class AppPage {
   waitUntilVisible() {
     const loginExists = ExpectedConditions.presenceOf(this.getLoginButton());
     const logoutExists = ExpectedConditions.presenceOf(this.getLogoutButton());
-    browser.wait(ExpectedConditions.or(loginExists, logoutExists), 5000);
+    browser.wait(ExpectedConditions.or(loginExists, logoutExists), 15000);
```

Comment:
Is it better to have a global timeout variable? 
Also, any reason a timeout of 5000 doesn't work?

Reply:
Actually, there is a global timeout, set in the protractor config file. i think this is passing a shorter one, for no reason i can tell. i ended up refactoring it and getting rid of the override anyway.